src/Pages/TemplatePage/TemplatePage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Outlet } from 'react-router-dom'
+import { Sidenav } from "../../Components/Sidenav/Sidenav"
+import "./templatepage.css"
+import { Navbar } from '../../Components/Navbar/Navbar'
```

Comment:
suggestion: create index file in components folder and export all components from it